### ISSUE TITLE
Examples of distinct attribute (API References)

### ISSUE THREAD

**Issue by curquiza** [meilisearch/documentation#491]
In the API Reference, the examples for the distinct attributes are introduced with `movie_id` which is the primary key of the index. The MeiliSearch hits are always distinct according to the unique identifier: so the primary key is the only attribute you will never put as a distinct attribute.
It does not help to understand the purpose of a distinct attribute 😇

- [ ] Change the [GET example](https://docs.meilisearch.com/references/distinct_attribute.html#get-distinct-attribute) response
- [ ] Change the [POST example](https://docs.meilisearch.com/references/distinct_attribute.html#update-distinct-attribute) input.

**Comment by ishika22** [comment]
Hi @curquiza , can you elaborate a little on the issue? If no one is working on it I would like to take it up

**Comment by curquiza** [comment]
Hello @ishika22 
The best would be to replace
- the `movie_id` used as distinct attributes by `skuid`
- the `movies` index UID by `shoes`

This to show the distinct attribute is not the unique identifier of the document
WDYT @meilisearch/docs-team?

**Comment by ishika22** [comment]
Sure @curquiza 
Can I take this up once it is confirmed?

**Comment by CaroFG** [comment]
Hi @ishika22! You can work on this 🙂 Thank you!

**Comment by guimachiavelli** [comment]
Agreed! @curquiza's suggestion is probably the best way of illustrating the main use-case for distinct attributes.

Thank you for the interest, @ishika22!
